0001-interledger-architecture/0001-interledger-architecture.md

```diff
@@ -18,9 +18,9 @@ The Interledger architecture is heavily inspired by the Internet architecture de
 
 ### Connectors
 
-A *connector* is a system that provides the service of forwarding Interledger packets towards the destination provided in the packet. Packets are forwarded across connections between senders, connectors and receivers, called *accounts*.
+A *connector* is a system that provides the service of forwarding Interledger packets towards the destination provided in the packet. Packets are forwarded across connections (called *accounts*) between senders, connectors and receivers.
```

Comment:
As described in https://github.com/interledger/rfcs/pull/364#discussion_r176349987